src/main/java/ro/jlg/staff/manager/application/dto/AddDepartmentDTO.java

```diff
@@ -0,0 +1,35 @@
+package ro.jlg.staff.manager.application.dto;
+
+public class AddDepartmentDTO {
+    private String name;
+    private String description;
+    private int maxEmployees;
+
+    // constructor
+    public AddDepartmentDTO(final String name, final String description,
```

Comment:
please remove commentaries and use @AllArgsConstructor and @Gettter instead of manually implementing them

Reply:
also, there is no need for setters here